generator/generator-core/src/main/kotlin/io/ia/ignition/module/generator/data/ModuleGeneratorContext.kt

```diff
@@ -97,6 +80,20 @@ class ModuleGeneratorContext(override val config: GeneratorConfig) : GeneratorCo
         }
     }
 
+    private fun buildDependencyEntries(scopes: List<ProjectScope>): Map<String, String> {
+        return mutableMapOf<String, String>().apply {
+            scopes.forEach { scope ->
+                TemplateMarker.dependencyKeyForScope(scope)?.let { tm ->
+                    this[tm.key] = DefaultDependencies.ARTIFACTS[scope]?.toDependencyFormat(config.buildDsl) ?: ""
+                    // if not a common scope and there is a common project, add it as a dependency to other scopes
+                    if (scope != COMMON && scopes.size > 1) {
```

Comment:
should this instead be something like `scope != COMMON && scopes.contains(COMMON)`? What happens if there are multiple scopes but _not_ a common one here?

Reply:
I don't think you can opt out of the common scopes if you configure a multi-scope project right now.  A common scope is a derived scope that is automatically added to any project that has 2+ scopes defined.